test/EventListener/JsonApiProblemExceptionListenerTest.php

```diff
@@ -75,10 +74,9 @@ public function it_runs_on_json_content_types(): void
         $listener = new JsonApiProblemExceptionListener($this->exceptionTransformer->reveal(), false);
         $this->request->getRequestFormat()->willReturn('html');
         $this->request->getContentType()->willReturn('application/json');
+        $listener->onKernelException($this->event);
 
-        $this->event->setResponse(Argument::type(JsonResponse::class))->shouldBeCalled();
-
-        $listener->onKernelException($this->event->reveal());
+        $this->assertNotNull($this->event->getResponse());
```

Comment:
The original was testing the instance of the returned value. Seems better to validate it similarly

Reply:
Fixed this, it now does an extra assert for to check the content of the event